scripts/install

```diff
@@ -83,19 +87,16 @@ fi
 # Create the php-fpm pool config
 POOL_FPM
 
-# Public access for curl
-ynh_app_setting_set $app unprotected_uris "/"
-
-# Relaod SSOwat configuration
-sudo yunohost app ssowatconf
-
-# Reload php5-fpm and Nginx
-sudo systemctl reload php5-fpm
-sudo systemctl reload nginx
+# Set ssowat config
+if [ $is_public -eq 0 ]
+then
+	ynh_app_setting_delete $app skipped_uris # Delete public access
+fi
 
-if [ "$is_public" = "No" ];
+if [ $is_public -eq 1 ]
 then
-	# Exit public access
-	ynh_app_setting_delete $app unprotected_uris
-	sudo yunohost app ssowatconf
+	ynh_app_setting_set $app skipped_uris "/" # Make app public if necessary
 fi
```

Comment:
Why changing this existing (tested) logic based on unprotected_uris to use skipped_uris (about which I'm very doubtful)?